XamlControlsGallery/ControlPages/ItemsRepeaterPage.xaml

```diff
@@ -212,14 +228,82 @@ the following attributes: Height, MaxHeight, Length, MaxLength, Diameter, and Ma
                 <muxc:ItemsRepeater x:Name="outerRepeater"
                                            VerticalAlignment="Top"
                                            ItemTemplate ="{StaticResource CategoryTemplate}">
-
                     <muxc:ItemsRepeater.Layout>
                         <muxc:StackLayout Orientation="Vertical" />
                     </muxc:ItemsRepeater.Layout>
-
                 </muxc:ItemsRepeater>
             </ScrollViewer>
         </local:ControlExample>
 
+        <local:ControlExample HeaderText="Animated Scrolling and Content Display"
+                              XamlSource="ItemsRepeater/ItemsRepeaterSample3_xaml.txt"
+                              CSharpSource="ItemsRepeater/ItemsRepeaterSample3_cs.txt">
+            <Grid>
+                <Grid.ColumnDefinitions>
+                    <ColumnDefinition Width="1*"/>
+                    <ColumnDefinition Width="1*"/>
+                </Grid.ColumnDefinitions>
+
+                <ScrollViewer x:Name="Animated_ScrollViewer" 
+                              Grid.Column="0"
+                              Height="175"
+                              Width="250"
+                              ViewChanging="Animated_ScrollViewer_ViewChanging">
+                    <muxc:ItemsRepeater x:Name="animatedScrollRepeater">
+                        <muxc:ItemsRepeater.ItemTemplate>
```

Comment:
nit: This line is technically not needed because it is the content property for ItemsRepeater.

Reply:
Cool, didn't know that! Fixed in upcoming commit.